session/session.go

```diff
@@ -5,11 +5,29 @@ import (
 	"sync"
 	"time"
 
+	"github.com/Shopify/sarama"
```

Comment:
You shouldn't import this just for the type. You should either use https://github.com/Sirupsen/logrus/blob/master/logrus.go#L82-94 or define your own minimal interface like Sarama does. Simply copy-paste the code.